conftest.py

```
@@ -940,23 +940,67 @@ def copyoffload_storage_secret(
+        "powerstore": [],  # Only basic credentials required
+        "infinibox": [],  # Only basic credentials required
+        "flashsystem": [],  # Only basic credentials required
+    }
+
+    # Add vendor-specific fields if configured
+    if storage_vendor in vendor_specific_fields:
+        for config_key, secret_key, required in vendor_specific_fields[storage_vendor]:
+            value = get_copyoffload_credential(config_key, copyoffload_cfg)
+            if value:
+                secret_data[secret_key] = value
+                LOGGER.info(f"✓ Added vendor-specific field: {secret_key}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)